packages/components/src/animate/stories/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * Internal dependencies
+ */
+import Animate from '../';
+import Notice from '../../notice';
+
+export default { title: 'Animate', component: Animate };
+
+export const _default = () => (
+	<Animate>
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>No default animation. Use one of type = appear, slide-in, or loading.</p>
+			</Notice>
+		) }
+	</Animate>
+);
+
+export const appear = () => (
+	<Animate type="appear">
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>Appear animation is meant for popover/modal content, such as menus appearing.</p>
+				<p>It shows the height and width of the animated element scaling from 0 to full size, from its point of origin.</p>
+			</Notice>
+		) }
+	</Animate>
+);
+
+export const loading = () => (
+	<Animate type="loading">
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>Loading animation is meant to be used to indicate that activity is happening in the background.</p>
+			</Notice>
+		) }
+	</Animate>
+);
+
+export const slideIn = () => (
+	<Animate type="slide-in">
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>Slide-in animation is meant for sidebars and sliding menus.</p>
+				<p>It shows the height and width of the animated element moving from a hidden position to its normal one.</p>
```

Comment:
I think that we should just focus on what the animation does. The use cases can be a lot more varied than sidebars and sliding menus.

We could also just avoid descriptions altogether for now until we have more thorough copy. The names are very self descriptive as is.

Reply:
Thanks for the reviews, I grabbed the content from the README, wasn't quite sure if we should include within.

Reply:
What do you think @ItsJonQ?

Reply:
@epiqueras I think some animation examples would be wonderful 😍 . I would be happy to help here you folks need!

> I grabbed the content from the README
@mkaz Unfortunately, the documentation example doesn't really do a lot 😅

Reply:
Yeah, we need better copy, or we can merge the variations without copy for now.

Reply:
Per this, I've removed the extra documentation and we can improve the README at the same time of what shows for the improved animation examples